### PR TITLE
Updated content for the intro description of E^0

### PR DESCRIPTION
**Pull request by juliannaweldon**
Changed wording slightly - less awkward and made it sound slightly more exciting. (hopefully)

### CODE REVIEW

File: src/components/about/index.js [hunk 1]
```diff
@@ -16,9 +16,11 @@ class About extends React.Component {
                 </StyledFeature>
                 <StyledMain>
                     <p>
-                        Week E<sup>0</sup> is the second best week of the year*. It’s our grand opening bash, so start off your year right by hanging out with your fellow first years for the
-                        first week full of back-to-back activities. The second week will be a little less crazy, but don’t miss the info session for E-Retreat or the chance to meet upper years. 
-                        Also don’t forget to come to BBQs and pancake breakfasts to meet faculty, student teams and clubs! Make friends, do something crazy, and have a lot of fun.
+                        Week E<sup>0</sup> is the second best week of the year*. Excited for September? Just wait til you see what the EUS has in store for you. 
+                        BBQs, movie nights, pancake breakfasts, and some crazy traditions to be a part of!It’s our grand opening bash, so start off
```
Comment (qgolsteyn) [inline src/components/about/index.js]: Small typo: space missing between "to be a part of!It's our grand opening"

File: src/components/about/index.js [hunk 2]
```diff
@@ -16,9 +16,11 @@ class About extends React.Component {
...
-                        first week full of back-to-back activities. The second week will be a little less crazy, but don’t miss the info session for E-Retreat or the chance to meet upper years. 
-                        Also don’t forget to come to BBQs and pancake breakfasts to meet faculty, student teams and clubs! Make friends, do something crazy, and have a lot of fun.
+                        Week E<sup>0</sup> is the second best week of the year*. Excited for September? Just wait til you see what the EUS has in store for you. 
+                        BBQs, movie nights, pancake breakfasts, and some crazy traditions to be a part of!It’s our grand opening bash, so start off 
+                        your year right by hanging out with your fellow first years for the first week full of back-to-back activities. The second 
+                        week will be a little less hectic, but don’t miss the info session for E-Retreat or the chance to interact with UBC’s various 
+                        engineering student teams and clubs.  We hope to see you around!
```
Comment (qgolsteyn) [inline src/components/about/index.js]: Double space before "We hope to see you around!"

File: src/components/about/index.js [hunk 3]
```diff
@@ -16,9 +16,11 @@ class About extends React.Component {
...
-                        Week E<sup>0</sup> is the second best week of the year*. It’s our grand opening bash, so start off your year right by hanging out with your fellow first years for the
-                        first week full of back-to-back activities. The second week will be a little less crazy, but don’t miss the info session for E-Retreat or the chance to meet upper years. 
-                        Also don’t forget to come to BBQs and pancake breakfasts to meet faculty, student teams and clubs! Make friends, do something crazy, and have a lot of fun.
+                        Week E<sup>0</sup> is the second best week of the year*. Excited for September? Just wait til you see what the EUS has in store for you. 
+                        BBQs, movie nights, pancake breakfasts, and some crazy traditions to be a part of!It’s our grand opening bash, so start off 
+                        your year right by hanging out with your fellow first years for the first week full of back-to-back activities. The second 
+                        week will be a little less hectic, but don’t miss the info session for E-Retreat or the chance to interact with UBC’s various
```
Comment (qgolsteyn) [inline src/components/about/index.js]: Thought: maybe let's not mention that the second week is less hectic?

**Review by qgolsteyn — CHANGES_REQUESTED** [review]

**Comment by juliannaweldon** [comment]
Made the changes requested.

**Comment by qgolsteyn** [comment]
Thanks!

**Pull request merged**